src/java/org/apache/cassandra/service/DataResolver.java

```diff
@@ -171,7 +171,14 @@ private PartitionIterator resolveWithReplicaFilteringProtection()
         // We need separate contexts, as each context has his own counter
         ResolveContext firstPhaseContext = new ResolveContext(count);
         ResolveContext secondPhaseContext = new ResolveContext(count);
-        ReplicaFilteringProtection rfp = new ReplicaFilteringProtection(keyspace, command, consistency, firstPhaseContext.sources);
+
+        ReplicaFilteringProtection rfp = new ReplicaFilteringProtection(keyspace,
+                                                                        command,
+                                                                        consistency,
+                                                                        firstPhaseContext.sources,
+                                                                        DatabaseDescriptor.getCachedReplicaRowsWarnThreshold(),
+                                                                        DatabaseDescriptor.getCachedReplicaRowsFailThreshold());
+
```

Comment:
Have you intentionally withdrew the caching of `UnaryOperator.identity()` below?

Reply:
Yes. See https://github.com/apache/cassandra/pull/659#discussion_r448037610.

Reply:
Oh, right, I missed that one